### PR TITLE
MCC-232377 use hybrid client

### PR DESCRIPTION
**Pull request by ncarroll-mdsol**
This PR updates the Babbage version, and extracts the version information into variables to make updating the Podspec easier.

### CODE REVIEW

**Comment by ncarroll-mdsol** [comment]
@venkatganga - when merged, this should fix the crash you were seeing.

**Comment by venkatganga** [comment]
:+1:

File: AppConnectSwift/Babbage.podspec [hunk 1]
```diff
@@ -10,13 +13,13 @@ end
 
 Pod::Spec.new do |s|
     s.name               = "Babbage"
-    s.version            = "2016.2.0.115"
+    s.version            = VERSION
```
Comment (kbohlmann-mdsol) [inline AppConnectSwift/Babbage.podspec:16]: Fresh.

**Comment by kbohlmann-mdsol** [comment]
🍰

**Pull request merged**